proseco/characteristics_guide.py

```diff
@@ -109,7 +109,8 @@
           {"Stage": 6,
            "SigErrMultiplier": 0,
            "ASPQ1Lim": 20,
-           "MagLimit": [5.0, 10.3],
+           # Final stage mag faint limit set by acceptance into candidate list
+           "MagLimit": [5.0, 20.0],
```

Comment:
I would figure that since the scaled dark cal used in guide star selection has only one temperature (the hot one right?) that the imposter checks might knock out the bonus stars.  Might just need an additional stage.